requirements/requirements-app.txt

```diff
@@ -25,7 +25,7 @@ Markdown<3.0
 marshmallow==3.14.1
 numpy==1.19.*
 openpyxl==2.4.7
-pandas==1.1.*
+pandas==1.3.*
```

Comment:
Why change the pandas version? We should look at specifically regression testing code that uses pandas.

Reply:
Was changed to silence more warnings that appear during tests. Not directly related to Django 3.2 so happy to revert this back if wanted.

Reply:
Looked at versions and some deprecations but mostly advanced features. Going forward with upgrade.